dbt_project.yml

```diff
@@ -1,11 +1,12 @@
 name: 'zendesk'
-version: '0.3.0'
+version: '0.3.1'
```

Comment:
is this a breaking change with the custom schemas?

Reply:
Errg yeah it would be. This was prior to us arriving that the final models will have their own schema. I should update this to version `0.4.0`